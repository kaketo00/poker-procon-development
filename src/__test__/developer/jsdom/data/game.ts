import type { Game } from '@/schema/game';

export const testGameInfoData: Game = {
  id: '10',
  status: 'progress',
  createdAt: '2024-04-15T18:52:10.296Z',
  players: {
    Player1: {
      name: 'Player1',
      status: 'active',
      point: 37600,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [
          {
            suit: 'Clubs',
            number: 1,
          },
          {
            suit: 'Diamonds',
            number: 7,
          },
          {
            suit: 'Spades',
            number: 6,
          },
          {
            suit: 'Spades',
            number: 9,
          },
          {
            suit: 'Clubs',
            number: 8,
          },
        ],
        hand: 1,
      },
    },
    Player2: {
      name: 'Player2',
      status: 'active',
      point: 17600,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [
          {
            suit: 'Hearts',
            number: 13,
          },
          {
            suit: 'Hearts',
            number: 1,
          },
          {
            suit: 'Spades',
            number: 5,
          },
          {
            suit: 'Clubs',
            number: 4,
          },
          {
            suit: 'Hearts',
            number: 4,
          },
        ],
        hand: 2,
      },
    },
    Player3: {
      name: 'Player3',
      status: 'active',
      point: 24800,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [
          {
            suit: 'Diamonds',
            number: 12,
          },
          {
            suit: 'Diamonds',
            number: 5,
          },
          {
            suit: 'Diamonds',
            number: 2,
          },
          {
            suit: 'Spades',
            number: 1,
          },
          {
            suit: 'Diamonds',
            number: 1,
          },
        ],
        hand: 2,
      },
    },
    Player4: {
      name: 'Player4',
      status: 'out',
      point: 0,
      round: {
        betPoint: 0,
        action: null,
        cards: [],
      },
    },
  },
  totalRound: 100,
  initialPoint: 20000,
  fee: 200,
  seatingOrder: ['Player3', 'Player4', 'Player1', 'Player2'],
  startedAt: '2024-04-15T18:52:13.832Z',
};
