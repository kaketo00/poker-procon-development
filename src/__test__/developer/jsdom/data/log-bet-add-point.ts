import type { Log } from '@/schema/log';

export const testLogBetAddPoint: Log = {
  round: 10,
  player: 'Player1',
  phase: 'bet-1',
  content: { type: 'bet', amount: 314, action: 'bet', payPoint: 314 },
  game: {
    id: '10',
    status: 'progress',
    createdAt: '2024-04-22T12:06:55.706Z',
    players: {
      Player1: {
        name: 'Player1',
        status: 'active',
        point: 64746,
        round: {
          betPoint: 314,
          first: 314,
          second: 0,
          action: 'bet',
          cards: [
            { suit: 'Clubs', number: 6 },
            { suit: 'Spades', number: 8 },
            { suit: 'Hearts', number: 9 },
            { suit: 'Diamonds', number: 1 },
            { suit: 'Spades', number: 1 },
          ],
        },
      },
      Player2: {
        name: 'Player2',
        status: 'active',
        point: 6256,
        round: {
          betPoint: 0,
          first: 0,
          second: 0,
          action: null,
          cards: [
            { suit: 'Diamonds', number: 6 },
            { suit: 'Clubs', number: 7 },
            { suit: 'Diamonds', number: 7 },
            { suit: 'Diamonds', number: 12 },
            { suit: 'Clubs', number: 1 },
          ],
        },
      },
      Player3: {
        name: 'Player3',
        status: 'out',
        point: 0,
        round: { betPoint: 0, first: 0, second: 0, action: null, cards: [] },
      },
      Player4: {
        name: 'Player4',
        status: 'active',
        point: 8084,
        round: {
          betPoint: 0,
          first: 0,
          second: 0,
          action: null,
          cards: [
            { suit: 'Diamonds', number: 3 },
            { suit: 'Hearts', number: 4 },
            { suit: 'Diamonds', number: 4 },
            { suit: 'Spades', number: 11 },
            { suit: 'Hearts', number: 11 },
          ],
        },
      },
    },
    totalRound: 100,
    initialPoint: 20000,
    fee: 200,
    seatingOrder: ['Player2', 'Player1', 'Player4', 'Player3'],
    startedAt: '2024-04-22T12:06:56.923Z',
  },
  pot: 914,
  minBetPoint: 314,
  createdAt: 1713787617161,
};
