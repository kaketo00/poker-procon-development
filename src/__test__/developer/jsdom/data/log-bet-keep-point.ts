import type { Log } from '@/schema/log';

export const testLogBetKeepPoint: Log = {
  round: 10,
  player: 'Player1',
  phase: 'bet-1',
  content: { type: 'bet', amount: -1, action: 'drop', payPoint: 0 },
  game: {
    id: '10',
    status: 'progress',
    createdAt: '2024-04-22T12:06:55.706Z',
    players: {
      Player1: {
        name: 'Player1',
        status: 'active',
        point: 64746,
        round: {
          betPoint: 0,
          first: 0,
          second: 0,
          action: 'drop',
          cards: [],
        },
      },
      Player2: {
        name: 'Player2',
        status: 'active',
        point: 6256,
        round: {
          betPoint: 0,
          first: 0,
          second: 0,
          action: null,
          cards: [
            { suit: 'Diamonds', number: 6 },
            { suit: 'Clubs', number: 7 },
            { suit: 'Diamonds', number: 7 },
            { suit: 'Diamonds', number: 12 },
            { suit: 'Clubs', number: 1 },
          ],
        },
      },
      Player3: {
        name: 'Player3',
        status: 'out',
        point: 0,
        round: { betPoint: 0, first: 0, second: 0, action: null, cards: [] },
      },
      Player4: {
        name: 'Player4',
        status: 'active',
        point: 8084,
        round: {
          betPoint: 0,
          first: 0,
          second: 0,
          action: null,
          cards: [
            { suit: 'Diamonds', number: 3 },
            { suit: 'Hearts', number: 4 },
            { suit: 'Diamonds', number: 4 },
            { suit: 'Spades', number: 11 },
            { suit: 'Hearts', number: 11 },
          ],
        },
      },
    },
    totalRound: 100,
    initialPoint: 20000,
    fee: 200,
    seatingOrder: ['Player2', 'Player1', 'Player4', 'Player3'],
    startedAt: '2024-04-22T12:06:56.923Z',
  },
  pot: 600,
  minBetPoint: 0,
  createdAt: 1713787617161,
};
