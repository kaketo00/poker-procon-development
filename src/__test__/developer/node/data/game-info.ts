import type { GameInfo } from '@/schema/game';

export const testGameInfoData: GameInfo = {
  currentRound: 1,
  phase: 'start',
  order: ['Player1', 'Player2', 'Player3', 'Player4'],
  pot: 0,
  minBetPoint: 0,
  players: {
    Player1: {
      name: 'Player1',
      status: 'active',
      point: 37600,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [],
      },
    },
    Player2: {
      name: 'Player2',
      status: 'active',
      point: 17600,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [],
      },
    },
    Player3: {
      name: 'Player3',
      status: 'active',
      point: 24800,
      round: {
        betPoint: 0,
        action: 'bet',
        cards: [],
      },
    },
    Player4: {
      name: 'Player4',
      status: 'out',
      point: 0,
      round: {
        betPoint: 0,
        action: null,
        cards: [],
      },
    },
  },
  totalRound: 100,
  initialPoint: 20000,
  fee: 200,
};
