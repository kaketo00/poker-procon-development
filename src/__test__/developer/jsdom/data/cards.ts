import type { Card, CardSet } from '@/schema/card';
import { Hand } from '@/schema/game';

type CardSetKye = 'A' | 'B1' | 'B2';
export const testCards: {
  [hand in Hand]: {
    [key in CardSetKye]: CardSet<Card>;
  };
} = {
  [Hand.RoyalStraightFlush]: {
    A: [
      { suit: 'Spades', number: 10 },
      { suit: 'Spades', number: 11 },
      { suit: 'Spades', number: 12 },
      { suit: 'Spades', number: 13 },
      { suit: 'Spades', number: 1 },
    ], // ロイヤルストレートフラッシュ(10♠ J♠ Q♠ K♠ A♠)
    B1: [
      { suit: 'Clubs', number: 10 },
      { suit: 'Clubs', number: 11 },
      { suit: 'Clubs', number: 12 },
      { suit: 'Clubs', number: 13 },
      { suit: 'Clubs', number: 1 },
    ], // ロイヤルストレートフラッシュ(10♣ J♣ Q♣ K♣ A♣)
    B2: [], // 使用しない
  },
  [Hand.StraightFlush]: {
    A: [
      { suit: 'Spades', number: 9 },
      { suit: 'Spades', number: 10 },
      { suit: 'Spades', number: 11 },
      { suit: 'Spades', number: 12 },
      { suit: 'Spades', number: 13 },
    ], // ストレートフラッシュ(9♠ 10♠ J♠ Q♠ K♠)
    B1: [
      { suit: 'Spades', number: 4 },
      { suit: 'Spades', number: 5 },
      { suit: 'Spades', number: 6 },
      { suit: 'Spades', number: 7 },
      { suit: 'Spades', number: 8 },
    ], // ストレートフラッシュ(4♠ 5♠ 6♠ 7♠ 8♠)
    B2: [
      { suit: 'Diamonds', number: 9 },
      { suit: 'Diamonds', number: 10 },
      { suit: 'Diamonds', number: 11 },
      { suit: 'Diamonds', number: 12 },
      { suit: 'Diamonds', number: 13 },
    ], // ストレートフラッシュ(9♦ 10♦ J♦ Q♦ K♦)
  },
  [Hand.FourOfAKind]: {
    A: [
      { suit: 'Clubs', number: 4 },
      { suit: 'Spades', number: 13 },
      { suit: 'Hearts', number: 13 },
      { suit: 'Diamonds', number: 13 },
      { suit: 'Clubs', number: 13 },
    ], // フォーカード(4♣ K♠ K♥ K♦ K♣)
    B1: [
      { suit: 'Spades', number: 7 },
      { suit: 'Hearts', number: 7 },
      { suit: 'Diamonds', number: 7 },
      { suit: 'Clubs', number: 7 },
      { suit: 'Clubs', number: 12 },
    ], // フォーカード(7♠ 7♦ 7♥ 7♣ Q♣)
    B2: [], // 使用しない
  },
  [Hand.FullHouse]: {
    A: [
      { suit: 'Spades', number: 12 },
      { suit: 'Hearts', number: 12 },
      { suit: 'Spades', number: 13 },
      { suit: 'Hearts', number: 13 },
      { suit: 'Diamonds', number: 13 },
    ], // フルハウス(Q♠ Q♥ K♠ K♠ K♦)
    B1: [
      { suit: 'Spades', number: 2 },
      { suit: 'Hearts', number: 2 },
      { suit: 'Spades', number: 3 },
      { suit: 'Hearts', number: 3 },
      { suit: 'Diamonds', number: 3 },
    ], // フルハウス(2♠ 2♦ 3♦ 3♠ 3♥)
    B2: [], // 使用しない
  },
  [Hand.Flush]: {
    A: [
      { suit: 'Spades', number: 2 },
      { suit: 'Spades', number: 6 },
      { suit: 'Spades', number: 9 },
      { suit: 'Spades', number: 13 },
      { suit: 'Spades', number: 1 },
    ], // フラッシュ(2♠ 6♠ 9♠ K♠ A♠)
    B1: [
      { suit: 'Clubs', number: 3 },
      { suit: 'Clubs', number: 4 },
      { suit: 'Clubs', number: 5 },
      { suit: 'Clubs', number: 7 },
      { suit: 'Clubs', number: 8 },
    ], // フラッシュ(3♠ 4♠ 5♠ 7♠ 8♠)
    B2: [
      { suit: 'Clubs', number: 2 },
      { suit: 'Clubs', number: 6 },
      { suit: 'Clubs', number: 9 },
      { suit: 'Clubs', number: 13 },
      { suit: 'Clubs', number: 1 },
    ], // フラッシュ(2♣ 6♣ 9♣ K♣ A♣)
  },
  [Hand.Straight]: {
    A: [
      { suit: 'Spades', number: 10 },
      { suit: 'Spades', number: 11 },
      { suit: 'Spades', number: 12 },
      { suit: 'Spades', number: 13 },
      { suit: 'Hearts', number: 1 },
    ], // ストーレート(10♠ J♠ Q♠ K♠ A♥)
    B1: [
      { suit: 'Spades', number: 4 },
      { suit: 'Spades', number: 5 },
      { suit: 'Spades', number: 6 },
      { suit: 'Spades', number: 7 },
      { suit: 'Hearts', number: 8 },
    ], // フラッシュ(4♠ 5♦ 6♥ 7♣ 8♦)
    B2: [
      { suit: 'Diamonds', number: 10 },
      { suit: 'Diamonds', number: 11 },
      { suit: 'Diamonds', number: 12 },
      { suit: 'Diamonds', number: 13 },
      { suit: 'Clubs', number: 1 },
    ], // フラッシュ(10♦ J♦ Q♦ K♦ A♣)
  },
  [Hand.ThreeOfAKind]: {
    A: [
      { suit: 'Spades', number: 5 },
      { suit: 'Spades', number: 6 },
      { suit: 'Spades', number: 8 },
      { suit: 'Hearts', number: 8 },
      { suit: 'Diamonds', number: 8 },
    ], // スリーカード(5♠ 6♠ 8♠ 8♥ 8♦)
    B1: [
      { suit: 'Spades', number: 5 },
      { suit: 'Spades', number: 6 },
      { suit: 'Spades', number: 7 },
      { suit: 'Hearts', number: 7 },
      { suit: 'Diamonds', number: 7 },
    ], // スリーカード(3♠ 4♠ 7♠ 7♥ 7♦)
    B2: [], // 使用しない
  },
  [Hand.TwoPair]: {
    A: [
      { suit: 'Spades', number: 3 },
      { suit: 'Spades', number: 5 },
      { suit: 'Hearts', number: 5 },
      { suit: 'Spades', number: 7 },
      { suit: 'Hearts', number: 7 },
    ], // ツーペア(3♠ 5♠ 5♥ 7♠ 7♥)
    B1: [
      { suit: 'Spades', number: 2 },
      { suit: 'Spades', number: 4 },
      { suit: 'Hearts', number: 4 },
      { suit: 'Spades', number: 6 },
      { suit: 'Hearts', number: 6 },
    ], // ツーペア(2♠ 4♠ 4♥ 6♠ 6♥)
    B2: [
      { suit: 'Hearts', number: 3 },
      { suit: 'Diamonds', number: 5 },
      { suit: 'Clubs', number: 5 },
      { suit: 'Diamonds', number: 7 },
      { suit: 'Clubs', number: 7 },
    ], // ツーペア(3♥ 5♦ 5♣ 7♦ 7♣)
  },
  [Hand.OnePair]: {
    A: [
      { suit: 'Spades', number: 6 },
      { suit: 'Hearts', number: 6 },
      { suit: 'Spades', number: 7 },
      { suit: 'Diamonds', number: 8 },
      { suit: 'Clubs', number: 9 },
    ], // ワンペア(6♠ 6♥ 7♠ 8♦ 9♣)
    B1: [
      { suit: 'Spades', number: 4 },
      { suit: 'Hearts', number: 4 },
      { suit: 'Spades', number: 9 },
      { suit: 'Diamonds', number: 10 },
      { suit: 'Clubs', number: 11 },
    ], // ワンペア(4♠ 4♥ 9♥ 10♦ J♣)
    B2: [
      { suit: 'Clubs', number: 6 },
      { suit: 'Diamonds', number: 6 },
      { suit: 'Diamonds', number: 7 },
      { suit: 'Clubs', number: 8 },
      { suit: 'Spades', number: 9 },
    ], // ワンペア(6♣ 6♦ 7♦ 8♣ 9♠)
  },
  [Hand.HighCard]: {
    A: [
      { suit: 'Diamonds', number: 2 },
      { suit: 'Spades', number: 4 },
      { suit: 'Clubs', number: 6 },
      { suit: 'Hearts', number: 7 },
      { suit: 'Spades', number: 1 },
    ], // ハイカード(2♦ 4♠ 6♣ 7♥ A♠)
    B1: [
      { suit: 'Clubs', number: 2 },
      { suit: 'Hearts', number: 4 },
      { suit: 'Diamonds', number: 6 },
      { suit: 'Clubs', number: 7 },
      { suit: 'Spades', number: 13 },
    ], // ハイカード(2♣ 4♥ 6♦ 7♣ K♠)
    B2: [
      { suit: 'Clubs', number: 2 },
      { suit: 'Hearts', number: 4 },
      { suit: 'Diamonds', number: 6 },
      { suit: 'Clubs', number: 7 },
      { suit: 'Hearts', number: 1 },
    ], // ハイカード(2♣ 4♥ 6♦ 7♣ A♥)
  },
  [Hand.Drop]: {
    A: [], // 使用しない
    B1: [], // 使用しない
    B2: [], // 使用しない
  },
};
