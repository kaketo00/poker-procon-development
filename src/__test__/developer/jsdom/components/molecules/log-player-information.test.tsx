import { ChakraProvider } from '@chakra-ui/react';
import { render } from '@testing-library/react';

import { testLogDraw } from '@/__test__/developer/jsdom/data/log-draw';
import type { LogPlayerInformationType } from '@/components/molecules/log-player-information';
import LogPlayerInformation from '@/components/molecules/log-player-information';
import theme from '@/styles/theme';

const Component = (props: LogPlayerInformationType) => (
  <ChakraProvider theme={theme}>
    <LogPlayerInformation {...props} />
  </ChakraProvider>
);

describe('LogPlayerInformation', () => {
  test('スナップショット/active - 強調あり', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player1',
          status: 'active',
          point: 19358,
          round: {
            betPoint: 442,
            first: 442,
            second: 0,
            cards: [
              { suit: 'Spades', number: 4 },
              { suit: 'Clubs', number: 9 },
              { suit: 'Spades', number: 10 },
              { suit: 'Spades', number: 11 },
              { suit: 'Hearts', number: 12 },
            ],
            action: 'bet',
          },
        }}
        isEmphasize
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/active - 強調なし', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player1',
          status: 'active',
          point: 19358,
          round: {
            betPoint: 442,
            first: 442,
            second: 0,
            cards: [
              { suit: 'Spades', number: 4 },
              { suit: 'Clubs', number: 9 },
              { suit: 'Spades', number: 10 },
              { suit: 'Spades', number: 11 },
              { suit: 'Hearts', number: 12 },
            ],
            action: 'bet',
          },
        }}
        isEmphasize={false}
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/all-in', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player1',
          status: 'active',
          point: 0,
          round: {
            betPoint: 442,
            first: 442,
            second: 0,
            cards: [
              { suit: 'Spades', number: 4 },
              { suit: 'Clubs', number: 9 },
              { suit: 'Spades', number: 10 },
              { suit: 'Spades', number: 11 },
              { suit: 'Hearts', number: 12 },
            ],
            action: 'all-in',
          },
        }}
        isEmphasize={false}
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/自分のターンではない', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player2',
          status: 'active',
          point: 19358,
          round: {
            betPoint: 442,
            first: 442,
            second: 0,
            cards: [
              { suit: 'Spades', number: 4 },
              { suit: 'Clubs', number: 9 },
              { suit: 'Spades', number: 10 },
              { suit: 'Spades', number: 11 },
              { suit: 'Hearts', number: 12 },
            ],
            action: 'bet',
          },
        }}
        isEmphasize={false}
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/out', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player1',
          status: 'out',
          point: 0,
          round: {
            betPoint: 0,
            first: 0,
            second: 0,
            cards: [],
            action: null,
          },
        }}
        isEmphasize={false}
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test('スナップショット/drop', () => {
    const { asFragment } = render(
      <Component
        player={{
          name: 'Player1',
          status: 'active',
          point: 19358,
          round: {
            betPoint: 422,
            first: 422,
            second: 0,
            cards: [],
            action: null,
          },
        }}
        isEmphasize={false}
        log={testLogDraw}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
