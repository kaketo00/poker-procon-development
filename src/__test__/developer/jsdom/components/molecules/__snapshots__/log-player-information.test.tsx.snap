// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogPlayerInformation スナップショット/active - 強調あり 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        One Pair → Three of a Kind
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-wxf7p2"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#7777"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-wnxf2w"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player1
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-1wh924d"
          >
            19,358
          </p>
          <p
            class="chakra-text css-eo8jpy"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`LogPlayerInformation スナップショット/active - 強調なし 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        One Pair → Three of a Kind
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-y0vblb"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#7777"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-1r2ihj4"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player1
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-1wh924d"
          >
            19,358
          </p>
          <p
            class="chakra-text css-eo8jpy"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`LogPlayerInformation スナップショット/all-in 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        One Pair → Three of a Kind
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-y0vblb"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#B4171E"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-1r2ihj4"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player1
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-12v1cak"
          >
            0
          </p>
          <p
            class="chakra-text css-1ow382j"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`LogPlayerInformation スナップショット/drop 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        One Pair → Three of a Kind
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-y0vblb"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#7777"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-1r2ihj4"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player1
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-1wh924d"
          >
            19,358
          </p>
          <p
            class="chakra-text css-eo8jpy"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`LogPlayerInformation スナップショット/out 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        One Pair → Three of a Kind
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-y0vblb"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#7777"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-1r2ihj4"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player1
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-1wh924d"
          >
            0
          </p>
          <p
            class="chakra-text css-eo8jpy"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;

exports[`LogPlayerInformation スナップショット/自分のターンではない 1`] = `
<DocumentFragment>
  <div
    class="css-8myd60"
  >
    <div
      class="css-19tvp9s"
    >
      <p
        class="chakra-text css-1876eo0"
      >
        High Card
      </p>
      <img
        alt="bubble"
        class="chakra-image css-13pr30p"
        src="/bubble.svg"
      />
    </div>
    <div
      class="css-70qvj9"
    >
      <div
        class="css-y0vblb"
      >
        <div
          class="css-15b5zx5"
        >
          <svg
            fill="none"
            height="72"
            viewBox="0 0 61 72"
            width="61"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12.4336 17.614C12.4336 7.88457 20.4352 0 30.3091 0C40.1829 0 48.1905 7.88457 48.1905 17.614C48.1905 27.3492 40.1829 35.2338 30.3091 35.2338C20.4352 35.2338 12.4336 27.3492 12.4336 17.614ZM30.4429 72.0003C42.2478 72.0003 52.931 67.3433 60.6312 59.8513V54.5366C60.6312 48.1099 55.2047 42.8941 48.505 42.8941H12.1262C5.42649 42.8941 0 48.1099 0 54.5366V59.5836C7.71835 67.2269 18.5046 72.0003 30.4429 72.0003Z"
              fill="#7777"
              fill-opacity="0.8"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-1r2ihj4"
      >
        <p
          class="chakra-text css-x7gufq"
        >
          Player2
        </p>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-5c5tbj"
        />
        <div
          class="css-gmuwbf"
        >
          <p
            class="chakra-text css-1wh924d"
          >
            19,358
          </p>
          <p
            class="chakra-text css-eo8jpy"
          >
            pt
          </p>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
